addon/globalPlugins/charinfo/__init__.py

```diff
@@ -32,7 +32,7 @@
 
 # Temporarily import private function as advised by Sean Budd from NVAccess.
 # A public API can be requested by add-on developers 2-3 month after 2022.3.3 release if no issue is found.
-from winAPI.sessionTracking import _isLockScreenModeActive
+from winAPI.sessionTracking import isLockScreenModeActive
```

Comment:
Importing this new symbol, the add-on will not work with older versions of NVDA (including 2023.1).

Could you use a try/except statement?

Also add a comment to indicate the versions of NVDA, e.g.:

```
try
    # For NVDA >= 2023.2
    import publicSymbol
except Blablabla:
    # For NVDA < 2023.1
    import _privateSymbol

```

This allows me to easily clean code when modifying the compatibility range.